README.md

```diff
@@ -6,8 +6,60 @@ spec](https://github.com/singer-io/getting-started/blob/master/SPEC.md).
 
 This tap: 
 
-- Pulls raw data from the [Google Ads API](https://developers.google.com/google-ads/api/docs/start). 
+- Pulls raw data from the [Google Ads API](https://developers.google.com/google-ads/api/docs/start).
```

Comment:
We should probably just say "Pulls data from..." instead of "raw" data given the field name / stream name transformations

Reply:
Updated the sentence